cctp_primitives/src/proving_system/verifier/certificate.rs

```diff
@@ -0,0 +1,52 @@
+use crate::{
+    type_mapping::{FIELD_SIZE, FieldElement, FieldHash},
+    proving_system::{
+        verifier::{UserInputs, ZendooVerifier},
+        error::ProvingSystemError,
+    },
+    utils::serialization::SerializationUtils
+};
+use primitives::FieldBasedHash;
+use crate::utils::get_wcert_sysdata_hash;
+
+/// All the data needed to reconstruct the aggregated input for the NaiveThresholdSignatureCircuit
+/// included in the Certificate.
+pub struct CertificateProofUserInputs<'a> {
+    curr_cumulative_sc_tx_comm_tree_root: &'a [u8; FIELD_SIZE],
```

Comment:
What is `curr_cumulative_sc_tx_comm_tree_root`?
Is it the `cumulative_sc_tx_comm_tree_root` for the last MC block of the withdrawal epoch `epoch_number`?

Reply:
Yes, please feel free to suggest a different name if you've already adopted some kind of convention on it